src/contracts/lookback_validation.py

```diff
@@ -0,0 +1,83 @@
+import numpy as np
+import pandas as pd
+import pyarrow as pa
+from datetime import datetime, timedelta
+from qablet_contracts.timetable import py_to_ts
+from qablet_contracts.timetable import TS_EVENT_SCHEMA
+from qablet.black_scholes.mc import LVMCModel
+
+# Importing the lookback_put_timetable function from lookback.py
+from src.contracts.lookback import lookback_put_timetable
+
+
+
+MC_PARAMS = {
+    "PATHS": 100_000,
+    "TIMESTEP": 1 / 250,
```

Comment:
for flat black scholes (such as in this case) you can make timestep large (such as 1000) with no loss of accuracy. Speed will be much faster.